code/game/objects/empulse.dm

```diff
@@ -4,7 +4,7 @@
 
 // #define EMPDEBUG 10
 
-proc/empulse(turf/epicenter, heavy_range, light_range, log=0)
+proc/empulse(turf/epicenter, heavy_range, light_range, list/exclude, log=FALSE)

```

Comment:
Do not change the signature like this. You might mess up calls to this function which expect logging to be the fourth argument. The signature has to be `/proc/empulse(turf/epicenter, heavy_range, light_range, log=FALSE, list/exclude = null)`